rust/src/write.rs

```diff
@@ -32,11 +33,10 @@ enum WriteMode<W: Write + Seek> {
     Attachment(AttachmentWriter<CountingCrcWriter<W>>),
 }
 
-pub enum PrivateRecordKind {
+#[derive(EnumSetType, Debug)]
+pub enum PrivateRecordOptions {
     /// The private record can appear in chunks.
```

Comment:
```suggestion
    /// If set and chunking is enabled, the private record will be written into a chunk. Otherwise, the record will be written directly to the file.
```